js/stex.js

```diff
@@ -1141,11 +1141,11 @@ module.exports = class stex extends Exchange {
             'side': side,
             'price': price,
             'stopPrice': stopPrice,
-            'amount': amount,
-            'cost': cost,
+            'amount': this.parseNumber (amount),
+            'cost': this.parseNumber (cost),
             'average': undefined,
-            'filled': filled,
-            'remaining': remaining,
+            'filled': this.parseNumber (filled),
+            'remaining': this.parseNumber (remaining),
```

Comment:
We don't need `parseNumber` here, we should be passing strings to `safeOrder` and `parseNumber` is done inside `safeOrder`.